journal/modernized/03-chapter-2.adoc

```diff
@@ -155,6 +155,8 @@ Sell all thou hast--Oft has this gracious word
 Within my bosom secretly been heard,
 To part with all.
 ____
+[.embedded-content-document.letter]
+--
 
 Ninth Month 2nd, 1833.
```

Comment:
This date actually needs a `[.signed-section-context-open]` above it. This isn't anything you did wrong, I think it must have been an oversight when @jaredh159 did the initial intake. This was probably Jared's very first asciidoc document so he was learning the ropes. :)

Reply:
Yup, Jason is correct, because this was such an early document, I failed to add that line. so it should read (line 161-ish):

```ASCIIDOC
[.signed-section-context-open]
Ninth Month 2nd, 1833.
```